apps/webClient/src/App.tsx

```diff
@@ -9,10 +9,51 @@ import KCollege from "./views/KCollege";
 
 type Props = {};
 
+const Monsterrat = { fontFamily: "Montserrat, sans-serif", fontWeight: 400 };
+const LibreBaskerville = {
+  fontFamily: "Libre Baskerville, sans-serif",
+  fontWeight: 400,
+};
+
+const BLACK = "#000000";
+const DARK_GREY = "#373733";
+
 const theme = createMuiTheme({
+  palette: {
+    text: {
+      primary: BLACK,
+      secondary: DARK_GREY,
+    },
```

Comment:
I'm not sure if we want to do this as MUI has a `black` color available by default (and grey) , but let's leave for now